readme.md

```diff
@@ -34,15 +34,16 @@ For questions and support, join the official [Ignite Discord](https://discord.gg
 
 Blockchains created with Ignite CLI use the [Cosmos SDK](https://github.com/cosmos/cosmos-sdk/) framework. To ensure the best possible experience, use the version of Ignite CLI that corresponds to the version of Cosmos SDK that your blockchain is built with. Unless noted otherwise, a row refers to a minor version and all associated patch versions.
 
-| Ignite CLI | Cosmos SDK   | IBC                  | Notes                                           |
-|------------|--------------|----------------------|-------------------------------------------------|
-| v0.23.0    | v0.45.5      | v3.0.1               | |
-| v0.21.1    | v0.45.4      | v2.0.3               | Supports Cosmos SDK v0.46.0-alpha1 and above |
-| v0.21.0    | v0.45.4      | v2.0.3               | |
-| v0.20.0    | v0.45.3      | v2.0.3               | |
-| v0.19      | v0.44        | v1.2.2               | |
-| v0.18      | v0.44        | v1.2.2               | `ignite chain serve` works with v0.44.x chains |
-| v0.17      | v0.42        | Same with Cosmos SDK | |
+| Ignite CLI  | Cosmos SDK    | IBC                     | Notes                                         |
+|-------------|---------------|-------------------------|-----------------------------------------------|
+ | v0.24.0     | v0.46.0       | v5.0.0                  |                                               |
```

Comment:
```suggestion
 | v0.24.0     | v0.46.1       | v5.0.0-rc0                  |                                               |
```

Reply:
Given the note above:

```
Unless noted otherwise, a row refers to a minor version and all associated patch versions.
```

I think it makes sense to keep them as they are.  For example `v5.0.0-rc0` of `ibc-go` will be compatible with all `v5.x.x` versions.

Reply:
:pray: